CHANGELOG.md

```diff
@@ -6,7 +6,11 @@ For more information about changelogs, check
 [Keep a Changelog](http://keepachangelog.com) and
 [Vandamme](http://tech-angels.github.io/vandamme).
 
-## 0.1.1  - 2017/06/29
+## 0.1.2  - 2017/07/11
+
+* [ENHANCEMENT] `Fb::Page#insights` now takes a hash of options on method call.
+
+## 0.1.1  - 2017/07/06
 
```

Comment:
I mentioned in the previous version that page#insights will be in the next path, so we will have a new patch. Previous version of page#insights used hard coded params.